lib/datawow/connection.rb

```diff
@@ -4,13 +4,14 @@
 module Datawow
   # :nodoc:
   class Connection
-    def initialize(model = nil)
+    def initialize(model = nil, version_api = '/api/v1')
```

Comment:
IMO, these arguments have to be defined explicitly with keyword arguments rather than positional arguments:

Example:
```ruby
class Connection
  def initialize(model:, version: 'v1')
```
and the client site
```ruby
Connection.new(model: :image_closed_question) #=> Got v1 as a default
```

Note: version api that client will supply better to be `v1` than `api/v1`

Reply:
This `def initialize(model:, version: 'v1')` very new to me. I must read more Ruby syntax (trips and tricks)

Reply:
This shit is really helpful to make sure we pass the right arguments.

ref: https://robots.thoughtbot.com/ruby-2-keyword-arguments